pkg/local-disk-manager/udev/device_test.go

```diff
@@ -0,0 +1,27 @@
+package udev
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestDevice_ParseDeviceInfo_UnPartition(t *testing.T) {
```

Comment:
these 2 test cases may be merged into one TestFunc, that may be better.

Reply:
ok, I try to modify it